src/ForecastDisplay.js

```diff
@@ -0,0 +1,38 @@
+import weatherDisplay from "./WeatherDisplay";
+import WeatherData from './WeatherData';
+
+//class used to display 4days forecast
+//to use it first declare API 
+//pass api reference and id of container in which you want to display forecast
+//to display forecast call function display(city) and pass name of the city as a string
```

Comment:
To czysto techniczny drobiazg, ale wielolinijkowe komentarze wygodniej pisać jako blokowe komentarze.